packages/docs/src/03-Usage.svench/20-knobs.md

```diff
@@ -0,0 +1,105 @@
+<script>
+   import { View } from 'svench';
+</script>
+
+<svench:options title="Knobs" />
+
+# Knobs
+
+Knobs are used to interact with the View components.
+
+## Defining and using knobs
+
+Knobs are defined for a `View` via the `knobs` prop, and their current values
+are retrieved via `let:knobs`.
+
+```svelte
+<View name="defining and using a knob, simple" knobs={{ someProp: "my default value" }} let:knobs>
+
+    <p>someProp: {knobs.someProp}</p>
+
+</View>
+
+<!-- equivalent, using deconstructors -->
+
+<View name="defining and using a knob, using deconstructor" 
+      knobs={{ someProp: "my default value" }} 
+      let:knobs={{someProp}}>
+
+    <p>someProp: {someProp}</p>
+
+</View>
+```
+
+<View name="defining and using a knob, simple" knobs={{ someProp: "my default value" }} let:knobs>
+
+    <p>someProp: {knobs.someProp}</p>
+
+</View>
+
+<View name="defining and using a knob, using deconstructor" knobs={{ someProp: "my default value" }} let:knobs={{someProp}}>
+
+    <p>someProp: {someProp}</p>
+
+</View>
+
+## Knobs passed as plain objects
+
+Knobs can be passed to the views as an object, where the keys are the knob
```

Comment:
Just style but I have done it this way everywhere else in the project... Paragraphs in Markdown are kept on a single line, and left for the IDE to wrap to its available width.

Reply:
Heh. I tend not to care very much about the raw paragraphs as it'll all be `<p>`ed by Markdown anyway. But your house, your rules. I'll try to remember to single line those paragraphs. :-)